sled-agent/src/instance_manager.rs

```diff
@@ -194,10 +207,9 @@ impl InstanceTicket {
         InstanceTicket { id, inner: Some(inner) }
     }
 
-    // (Test-only) Creates a null ticket that does nothing.
+    // Creates a null ticket that does nothing.
```

Comment:
Yea... the only reason I added this was because I couldn't destructure `running_state` to get the existing ticket in `Instance::migrate` and so I just swap it with this "null" ticket. The resulting ticket is never actually stored on an instance.

Reply:
Don't need this anymore, undone.